packages/datatrak-web-server/src/__tests__/utils/processSurveyResponse.test.ts

```diff
@@ -314,7 +319,10 @@ describe('processSurveyResponse', () => {
         {
           question_id: 'question1',
           type: QuestionType.Entity,
-          body: 'answer1',
```

Comment:
I think that this should still be a string answer right? I don't think that central server handles body's that are an Entity object?